microprofile.ls/README.md

```diff
@@ -112,3 +112,26 @@ To add external snippets (like Quarkus snippets) an implementation of `ISnippetR
  * [Java Quarkus snippets loader](https://github.com/redhat-developer/quarkus-ls/tree/master/quarkus.ls.ext/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/snippets).
  * [JSON Quarkus snippet](https://github.com/redhat-developer/quarkus-ls/tree/master/quarkus.ls.ext/com.redhat.quarkus.ls/src/main/resources/com/redhat/quarkus/snippets).
  * Java Quarkus snippets loader must be declared in [META-INF/services/org.eclipse.lsp4mp.ls.commons.snippets.ISnippetRegistryLoader](https://github.com/redhat-developer/quarkus-ls/blob/master/quarkus.ls.ext/com.redhat.quarkus.ls/src/main/resources/META-INF/services/org.eclipse.lsp4mp.ls.commons.snippets.ISnippetRegistryLoader) 
+
+ Manage complex properties
+-------
+
+The available properties in `microprofile-config.properties` comes from the external component (ex: MicroProfile JDT LS extension). In some case a property 
+cannot be computed on the external component and must be computed on MicroProfile LS side.
+
+A usecase for that, is the MicroProfile Reactive Messaging. Some properties are computed according a property value declared in the `microprofile-config.properties`. 
```

Comment:
An example of this is the MicroProfile Reactive Messaging properties.

Reply:
fixed